src/commands.cpp

```diff
@@ -7929,19 +8020,19 @@ CommandGetRegisteredContacts::CommandGetRegisteredContacts(MegaClient* client, c
 }
 
 void CommandGetRegisteredContacts::procresult()
-{
-    processResult(*client->app, client->json);
-}
-
-void CommandGetRegisteredContacts::processResult(MegaApp& app, JSON& json)
 {
     Error e;
-    if (checkError(e, json))
+    if (checkError(e, client->json))
```

Comment:
I've just moved the error's handling to the non-static method. The static `processResult()` used by unit-tests only aims to test the JSON response, not the error cases.